src/main/resources/db/migration/V5_0__change_of_tags_structure.sql

```diff
@@ -0,0 +1,74 @@
+-- Create new tables
+create table category_tag (
+    id bigint not null auto_increment,
+    created datetime(6),
+    display_name varchar(255),
+    modified datetime(6),
+    municipality_id varchar(255) not null,
+    name varchar(255) not null,
+    namespace varchar(255) not null,
+    primary key (id)
+) engine=InnoDB;
+
+create table external_id_type_tag (
```

Comment:
external_id_type_tag is modeled exactly like status_tag. 
category_tag is almost identical.

Why separate tables?

Reply:
CategoryTag has a relationship to TypeTag while ExternalIdTypeTag and StatusTag does not have any relationships, hence I chose to separate CategoryTag and TypeTag as own entities. I thought about using the same model for ExternalIdTypeTag and StatusTag, but decided to have them as separate tables/repositories. That is of course just my personal taste, I can change the model if desired. Right now the model looks like this (which was verified to be ok with the rest of the team (except you who was on vacation) last week):

![image](https://user-images.githubusercontent.com/56998189/225552942-f002f08c-1224-49c2-b6ae-24a993493cee.png)

Reply:
Vacation or not: 
If you see yourself duplicating code, something is wrong with the design.

Reply:
Sure, I'll refactor the model so the StatusTag and the ExternalIdTypeTag shares the same table.

Reply:
Maybe remove suffix?